app/components/users/UserAdd.js

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import toastr from 'toastr';
+import * as userActions from '../../actions/userActions';
+import UserForm from './UserForm';
+
+class UserAdd extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.state = {
+            user: Object.assign({}, props.user),
+            saving: false
+        };
+
+        this.handleSave = this.handleSave.bind(this);
+    }
+
+    handleSave(user) {
+        this.setState({ saving: true });
+        this.props.actions.createUser(user)
+            .then(() => {
+                toastr.success('User created successfully');
+                this.setState({ saving: false });
```

Comment:
I would like here to be redirected to /users list if you want it. 

You need to do this import: `import { browserHistory } from 'react-router';`

And do the redirection: `browserHistory.push('/app/users');``
And also in the edit component, after the execution.

What do you think?